src/vma/util/sys_vars.h

```diff
@@ -677,6 +677,7 @@ extern mce_sys_var & safe_mce_sys();
 #define FLOW_STEERING_MGM_ENTRY_SIZE_PARAM_FILE		"/sys/module/mlx4_core/parameters/log_num_mgm_entry_size"
 #define VIRTUAL_DEVICE_FOLDER			"/sys/devices/virtual/net/%s/"
 #define BOND_DEVICE_FILE				"/proc/net/bonding/%s"
+#define CX4_DRIVER_PATH					"/sys/class/net/%s/device/driver/module/drivers/pci:mlx5_core"
```

Comment:
change define name from `CX4_DRIVER_PATH` to `MLX5_DRIVER_PATH`

1. this is more descriptive of the path 
2. today this is CX4 tomorrow mlx5 will also handle CX5